INSTALL.md

```diff
@@ -0,0 +1,115 @@
+# Building the AWS OFI NCCL Plugin
+
+We strongly recommend starting with a release tarball available on the
+[GitHub Release Page](https://github.com/aws/aws-ofi-nccl/releases) when
+building from source for production uses..
+
+`aws-ofi-nccl` requires a working installation of Libfabric. You can
```

Comment:
we should specify a version here.  And note that the version required is different on EFA than other providers.